app/src/main/java/com/kamron/pogoiv/MainActivity.java

```diff
@@ -253,6 +250,7 @@ public void onClick(View v) {
 
 
         displayMetrics = this.getResources().getDisplayMetrics();
+        initOCR();
```

Comment:
This call uses `candyOrder` before it's initialized correctly. Fixed in c9521d0.